docs/README.md

```diff
@@ -327,6 +333,59 @@ the application with the selected profile.
 ```shell
 export AWS_PROFILE=other_profile
 ```
+#### Azure Provider Prerequisites
+
+This step is optional, in the case you want to use the Azure Provider. 
+
+> **Note**:
+>
+> Booster is free to use, but notice that the resources deployed to your cloud provider
+> might generate some expenses.
+>
+> In Azure, when you're not eligible for the free tier, resources are charged on-demand. Deploying a
+> Booster project and sending a few commands and queries should cost just a few cents.
+>
+> In any case, make sure to un-deploy your application with the command `boost nuke -e production`
+> if you're not planning to keep using it.
+
+Now it is a good time to create that Microsoft Azure account. You can do so from [the Microsoft SingUp page](https://signup.live.com/). Then you can go to your Azure account in this [link](https://azure.microsoft.com).
+
+Once you have created the Azure account, you need to install the [Azure CLI](https://docs.microsoft.com/es-es/cli/azure/install-azure-cli) in your machine, by following the instructions of the Azure CLI documentation.
+
+Apart from that, you need to install [jq](https://stedolan.github.io/jq/download/) in your system.
+
+The final step consists of signing in in your Azure account from the Azure CLI:
+
+```shell
+az login
+```
+
+You can check the different login options in this [reference document](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli). 
+
+Once you are logged in from the CLI, you can deploy applications with Booster.
+
+> You can see more details in [Providers configuration](#providers-configuration) section.
+
+#### Kubernetes Provider Prerequisites
+
+This step is optional, in the case you want to use the Kubernetes provider. 
```

Comment:
Why are these steps optional? if you want to run the app on one of these providers you have to do these steps so they are not optional

Reply:
I copied the AWS part which is also optional. These are the prerequisites to deploy an application and you don't need to set up all providers to go. Would it be better to say "This step is only required if you want to work with K8s provider" ? @rdiaz82